### DISCUSSION TITLE
<title>How can I start contributing to open-source projects?</title>

### DISCUSSION THREAD

**Discussion by abdullahamarmusa** [community/community#109571] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Explore beginner-friendly projects on GitHub and start with small tasks like documentation updates or bug fixes.

**Comment by johnmillariu** [comment]
A great way to start contributing to open-source projects is by exploring platforms like GitHub, where you can find beginner-friendly projects. Begin by identifying projects aligned with your interests and skills, then look for labeled issues marked as 'good first issue' or 'beginner-friendly.' These tasks often involve simple fixes like documentation updates or bug squashing. Don't hesitate to reach out to project maintainers or community members for guidance, and remember that every contribution, no matter how small, makes a valuable impact on the open-source community.

**Comment by ettaboyle** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
